docs/authoring.md

```diff
@@ -1,49 +1,51 @@
 # Authoring Components
 
 ## Overview
-**Authoring Support is still in preview**
 
-C#/WinRT is working to provide support for authoring Windows Runtime components. You can write a library in C#, and specify that it is a `CsWinRTComponent` for C#/WinRT to produce a WinMD that any WinRT compatible language can use. For example, a library written in C# can be used by a C++ program, via C#/WinRT and C++/WinRT.
+**Note: Authoring Support is still in preview**
 
+C#/WinRT provides support for authoring Windows Runtime components. You can write a library in C#, and specify that it is a `CsWinRTComponent` for C#/WinRT to produce a WinMD that any WinRT compatible language can use. For example, a library written in C# can be used by a C++ program, via C#/WinRT and C++/WinRT.
 
 ## References
 Here are some resources that demonstrate authoring C#/WinRT components and the details discussed in this document.
-1. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringTest
+1. [Simple C#/WinRT component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo) and associated [walkthrough](https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/create-windows-runtime-component-cswinrt) on creating a C#/WinRT component and consuming it from C++/WinRT
 
-2. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringConsumptionTest
-
-3. https://github.com/AdamBraden/MyRandom
+2. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringTest
 
+3. https://github.com/microsoft/CsWinRT/tree/master/src/Tests/AuthoringConsumptionTest
 
 ## Authoring the C# Component
 To create a library, select the Class Library (.NET Core) template in Visual Studio. C#/WinRT projects require Windows API version specific .NET frameworks.
 
-Accepted `<TargetFramework>` properties |
+Accepted `<TargetFramework>` versions |
 --- |
 `net5.0-windows10.0.19041.0` |
 `net5.0-windows10.0.18362.0` |
 `net5.0-windows10.0.17763.0` |
 
 The library you are authoring should specify the following properties in its project file: 
+
 ``` csproj
 <PropertyGroup>
   <!-- update the Windows API version to reflect your TargetFramework -->

```

Comment:
"Choose your TargetFramework for the desired Windows SDK projection"

or just remove the comment if you think that's better